src/js/clinvar/eva-clinical-widget-panel.js

```diff
@@ -146,7 +149,7 @@ EvaClinicalWidgetPanel.prototype = {
                     header: {
                         baseCls: 'eva-header-1'
                     },
-                    title: 'ClinVar Browser <span class="icon icon-generic title-header-icon" data-icon="i" data-qtip="Search ClinVar (release 03-2015) using any combination of the filtering options on the left hand-side. Search results can be exported in CSV format and individual variants can be further investigated using the in-depth ClinVar Data tabs found below the main results table." style="margin-bottom:0px;"></span>',
+                    title: 'Variants found <span class="icon icon-generic title-header-icon" data-icon="i" data-qtip="Search ClinVar (release 03-2015) using any combination of the filtering options on the left hand-side. Search results can be exported in CSV format and individual variants can be further investigated using the in-depth ClinVar Data tabs found below the main results table." style="margin-bottom:0px;"></span>',
```

Comment:
The `data-qtip` field is no longer necessary since everything is explained at the top paragraph.